Index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>mon cv</title>
+    </head>
+
+    <body>
+        <h1>Blaise De Carvalho</h1>
+        <p><a href="photo_cv.jpg"><img src="photo_mini_cv.jpg"></a>
+            
+        </p>
+    	   <p>+337 82 91 95 91</br>
+            <a href="mailto:blaise@gmail">Contact me by email</a></br>
+            <a href="https://www.linkedin.com/in/blaisedecarvalho/">Visit my Linkedin profile</a></br>
+            Carrer de les Carolines 7, 2/2° Barcelona, Spain</br>
+            French</br>
+            N.I.E: Y6216347-J</p>
+
+    <p> <a href="#Education">Education</a></br>
+        <a href="#Professional">Professional Experience</a></br>
+        <a href="#Skills">Skills</a></br>
+        <a href="#Interests">Personal Interests</a></p>          
+
+            <p>Currently working as a Lodging sales</br>
+            specialist at eDreams Odigeo. I previously</br>
+            worked for three years as Product Manager</br>
+            Portugal and Specialist in Iberian Peninsula,</br>
+            creating and selling tailor-made travels and
+            itineraries.</p>
+
+            <p>Today I would like to join a new team in which</br>
+            I could utilize my strong selling skills, my</br>
+            hospitality network in Portugal and Spain, my</br>
+            important sense for service and my deep</br>
+            relational sensitivity.</p>    
+        
+        <h2><a id="Education"></a>Education</h2>    
+            <ul>
+                <li><a href="https://www.ulisboa.pt/en">Universidade de Lisboa – ESHTE</a></br>
+                    Master degree in Tourism and Communication</br>
+                    2012-2014</li>
+                <li><a href="https://www.parisnanterre.fr/portail-institutionnel-693762.kjsp">Université Paris X Nanterre</a></br>
+                    Bachelor’s Degree in Geography and Spatial Planning</br>
+                    Erasmus – Instituto de Geografia e Ordenamento do Territorio - Lisbon</br>
+                    2009-2012</li>    
+            </ul>            
+
+        <h2><a id="Professional"></a>Professional Experience</h2>
+            <h4>From Home SARL</h4>  
+                    <p><b>Full Stack Dev</b>
+            <ul>
+                <li>Break some mouths on Python</li>
+                <li>Smooth swagism on JavaScript</li>
+            </ul>
+            <p><i>Recommandations</i> <img src="doigt.png"> <a href="https://www.linkedin.com/in/gabriel-raty%C3%A9-9b270057/">Gabriel Ratyé</a></p>            
```

Comment:
- La balise `i`tu vas l'oublier assez rapidement dès lors que tu vas utiliser du css. Garde bien à l'esprit que 
 concernant le html, tu ne dois pas te soucier de l'apparence. Donc tu utiliseras probablement une autre balise (genre un span) avec une classe qui te permettra de mettre l'italique avec le css. Mais pour l'instant pas de css donc ça peut passer.
- Sache qu'il y a aussi une balise `em` (pour "emphasize") qui a le même effet, mais est plus intéressante car elle joue un rôle sémantique (en gros c'est bon pour le référencement, tu indique à google que ce qui est contenu dans cette balise est important).
- Même remarque pour la balise `b` (que tu utilises plus bas) et la balise `strong` qui met en gras (mais ça on s'en fou) et donne une information sur le contenu.
- Pour résumer, le HTML sert à baliser et à définir la valeur sémantique de ton contenu. C'est comme ça que tu choisis les balises que tu utilises. Les balises `i` et `b` en général on ne les utilise pas.
- ça veut pas dire que ça ne marche pas, tu verras probablement ça utilisé ailleurs, mais c'est pas considéré comme une bonne pratique, et ça apporte pas grand chose.

Reply:
C'est noté chef ! Très clair !